examples/assured-workloads-example/main.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "assured_workloads" {
+  source = "../../assured-workloads-example"
```

Comment:
@erlanderlo because kms-solutions repo is not published in [terraform.registry.io](terraform.registry.io) we can't use the standard `terraform-google-modules/module/google` or`GoogleCloudPlatform/module/google` approach here.

For all the TF modules in kms-solutions, we are using the local module source (with `../../`) approach.

If we want, a possibility to pull the AW module remotely from kms-solutions would be to use the GitHub tag (e.g: `source = "git::ssh://git_org/git_repo.git?ref=v1.0.0"`), but I believe we should keep it as is for now, while the module isn't 100% implemented (this is a partial PR) and update it later, if it makes sense!
WDYT? 

FYI @akieras